### PR TITLE
[STABLE-2112]: add example script

### PR DESCRIPTION
**Pull request by hsinghgrewal**
# Summary
Add a example script to get started with CCTP. The script use web3.js transfer USDC from eth testnet address to avax testnet address.

### CODE REVIEW

**Review by walkerq — COMMENTED** [review]
Looks good! Just a few minor comments.

**Review by walkerq — DISMISSED** [review]
🔥

**Review by chasemcdermott — COMMENTED** [review]
Looks awesome, couple small things!

**Review by walkerq — APPROVED** [review]

**Review by chasemcdermott — APPROVED** [review]

File: examples/index.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    const destinationAddressInBytes32 = await ethMessageContract.addressToBytes32(mintRecipient);
+    const AVAX_DESTINATION_DOMAIN = 1;
+    
+    // STEP 0: Approve messenger contract to withdraw from our active metamask address
+    // const approveTx = await usdcEthContract.approve(TOKEN_MESSENGER_CONTRACT_ADDRESS, 500000) // 0.5 USDC
+    // console.log(`ApproveTx: ${approveTx.hash}`)
+    
+    // Wait for transaction to complete
+    // const approveTxReceipt = await provider.waitForTransaction(approveTx.hash)
+    // console.log(`ApproveTxReceipt: ${approveTxReceipt}`)
```
[the remark this answers is not in the document]
Comment (swhidby1985) [reply examples/index.js]: > ?



> Metamask ask before burn tx even if we approve

**Pull request merged**